src/libstd/env.rs

```diff
@@ -182,6 +182,12 @@ impl fmt::Debug for VarsOs {
 /// * Environment variable is not present
 /// * Environment variable is not valid unicode
 ///
+/// # Panics
+///
+/// This function may panic if `key` is empty, contains an ASCII equals sign
```

Comment:
Another question: the function documentation says that when `key` is not present, an error is returned, not a panic. So is the "is empty" here correct?